scripts/fast-reboot-filter-routes.py

```diff
@@ -22,7 +21,7 @@ def get_connected_routes():
             sys.exit(ret)
         if output is not None:
             route_info = json.loads(output)
-            for route, info in route_info.items():
+            for route in route_info.keys():
```

Comment:
Will this work? I though keys() in Python3 returns type `dict_keys` that can't be iterated upon? I think you need to force convert this to list: `list(route_info.keys())`